Resources/Prototypes/_SpaceCats/Entities/Objects/Devices/flatpack.yml

```
@@ -0,0 +1,8 @@
+- type: entity
+  parent: BaseFlatpack
+  id: BookPrinterFlatpack
+  name: book printer flatpack
+  description: A flatpack used for constructing a book printer.
+  components:
+  - type: Flatpack
+    entity: BookPrinter
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)